### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kgaonkar6**
#### What scientific question is your analysis addressing?
Just adding a suggestion to add bash script per each stacked PR for the reviewer to test/run and will also be updated in the CI script ( once we have test data )

#### What was your approach?
Just added info about adding a bash script in CONTRIBUTING.md

### CODE REVIEW

**Review by runjin326 — APPROVED** [review]
Thanks for the update!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,12 +103,13 @@ git stash pop
...
 # add, stash, commit, push
 git add analyses/new-analyses/02-second-script.R
+git add analyses/new-analyses/run-analysis.sh
 git stash --include-untracked --keep-index
 git commit -m "Add second script for new analysis"
 git push origin new-analysis-second
 ```
 
-These steps can be repeated for `03-third-script.R`.
+These steps can be repeated for `03-third-script.R` and updated `run-analysis.sh` at each step.
```
Comment (jharenza) [inline CONTRIBUTING.md]:
```suggestion
These steps can be repeated for `03-third-script.R` and `run-analysis.sh` can be updated at each step.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -74,11 +75,12 @@ git checkout -b new-analysis-first new-analysis
 git reset master
 ```
 
-Now we're ready to add, commit, and push `01-first-script.R`.
+Now we're ready to add, commit, and push `01-first-script.R` and the bash script `run-analysis.sh` with the information to run `01-first-script.R`
```
Comment (jharenza) [inline CONTRIBUTING.md]:
```suggestion
Now we're ready to add, commit, and push `01-first-script.R` and the bash script `run-analysis.sh` with the information to run `01-first-script.R`.
```

**Review by jharenza — APPROVED** [review]
few minor comments, but looks good

**Pull request merged**